src/zenml/new/pipelines/pipeline.py

```diff
@@ -704,32 +733,19 @@ def _run(
             finally:
                 constants.SHOULD_PREVENT_PIPELINE_EXECUTION = False
 
-            runs = Client().list_pipeline_runs(
-                deployment_id=deployment_model.id,
-                sort_by="desc:start_time",
-                size=1,
-            )
-
-            if runs.items:
-                self.register_running_versions(new_version_requests)
-                run_url = dashboard_utils.get_run_url(runs[0])
-                if run_url:
-                    logger.info(f"Dashboard URL: {run_url}")
-                else:
-                    logger.info(
-                        "You can visualize your pipeline runs in the `ZenML "
-                        "Dashboard`. In order to try it locally, please run "
-                        "`zenml up`."
-                    )
+            self.register_running_versions(new_version_requests)
```

Comment:
on another note, this should probably be renamed to `register_running_model_versions` @avishniakov

Reply:
This method is removed from develop already - no need.